lib/Hero/Hero.js

```diff
@@ -112,9 +82,26 @@ Hero.propTypes = {
    */
   slides: PropTypes.arrayOf(
     PropTypes.shape({
-      title: PropTypes.string,
-      subTitle: PropTypes.string,
-      imageUrl: PropTypes.string,
+      /**
+       * Custom Image optimizer component
+       * e.g. `import Image from 'next/image'`
+       */
+      imageComponent: PropTypes.elementType,
+      /**
+       * Title of Hero component
+       * e.g. `Hello world`
+       */
+      title: PropTypes.string.isRequired,
+      /**
+       * Subtitle of Hero component
+       * e.g. `How to make you feel good`
```

Comment:
I appreciate the effort, but I don't think we necessarily need these examples. The names of the props are self-explanatory and the types provide enough info as to what they should be.